pkg/operator/controller/ingress/controller.go

```diff
@@ -501,31 +501,40 @@ func filterTLS13Ciphers(ciphers []string) []string {
 // the finalizer.
 func (r *reconciler) ensureIngressDeleted(ingress *operatorv1.IngressController) error {
 	errs := []error{}
-	if err := r.finalizeLoadBalancerService(ingress); err != nil {
-		errs = append(errs, fmt.Errorf("failed to finalize load balancer service for %s/%s: %v", ingress.Namespace, ingress.Name, err))
+	svc, err := r.finalizeLoadBalancerService(ingress)
```

Comment:
Shouldn't read `svc` is `err` is not nil... how about just returning a "service still exist" error and when the service is not found return no error?

Reply:
`retryable.New` could be used for such resource-still-exists errors.